package.json

```diff
@@ -57,7 +57,13 @@
     "grunt-prompt": "~1.3.0",
     "grunt-replace": "0.6.2",
     "grunt-sass": "~1.1.0-beta",
+    "gulp": "3.8.10",
+    "gulp-iconfont": "1.0.0",
+    "gulp-iconfont-css": "0.0.9",
+    "gulp-install": "0.2.0",
+    "gulp-sketch": "0.0.7",
     "jquery": "2.1.4",
+    "jsdoc": "^3.4.0",
```

Comment:
At first I can see an issue with documentationjs and it is it does not document classes that jsdoc does (see: https://github.com/CartoDB/cartodb.js/blob/widgets-jsdoc/src/core/view.js#L7 or https://github.com/CartoDB/cartodb.js/blob/widgets-jsdoc/src/geo/ui/widgets/widget-view.js#L6).

Jsdoc documents the class because the comments on the `constructor` method. It is not a great problem and can be solved adding the `@class` to the class. In addition I see there are no commnets from `@classdesc` tag.